packages/talos_agent/src/messaging/api.rs

```diff
@@ -67,9 +76,8 @@ pub struct DecisionMessage {
     #[serde(skip_deserializing)]
     pub suffix_start: u64,
     pub version: u64,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub safepoint: Option<u64>,
-    #[serde(skip_serializing_if = "Option::is_none")]
+    pub conflicts: Option<Vec<ConflictMessage>>,
```

Comment:
I have updated this on `certifier` end to pass `conflict_version: Option<u64>`. Could you sync that here?